app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -23,13 +23,63 @@ To learn more about the information in this table, see the [matching docs](/docs
 
 ## Configuration
 
+{% if_version lte:2.4.x %}
 ### LocalityAwareness
 
 Locality-aware load balancing is enabled by default unlike its predecessor [localityAwareLoadBalancing](/docs/{{ page.version }}/policies/locality-aware).
 
 - **`disabled`** – (optional) allows to disable locality-aware load balancing. When disabled requests are distributed 
 across all endpoints regardless of locality.
 
+{% endif_version %}
+{% if_version gte:2.5.x %}
+### LocalityAwareness
+Locality-aware load balancing provides a more robust and straightforward method for balancing traffic within and across zones. This not only allows you to route traffic across zones when the local zone service is unhealthy but also enables you to define traffic prioritization within the local zone and set cross-zone fallback priorities.
+
+#### Default behaviour
+Locality-aware load balancing is enabled by default, unlike its predecessor [localityAwareLoadBalancing](/docs/{{ page.version }}/policies/locality-aware). Requests are distributed across all endpoints within the local zone first unless there are not enough healthy endpoints.
+
+#### Disabling cross zone routing
+It's possible to disable cross-zone traffic. In this case, all endpoints are treated equally.
```

Comment:
Can we add how to do this here?

Reply:
What you are actually disabling is localityAware LoadBalancing so this title and the first sentence is wrong. 

#### Disabling locality aware routing

If you do so, all endpoints regardless of their zone will be treated equally. To do this do:

```yaml
localityAwareness:
  disabled: true
```